### ISSUE TITLE
Incompatible with "org.apache.httpcomponents:httpclient:4.5.7"

### ISSUE THREAD

**Issue by Paramood** [aws/aws-sdk-java-v2#1168]
AWS SDK is incompatible with "org.apache.httpcomponents:httpclient:4.5.7" which is added by Spring Boot gradle plugin v.2.1.x. The exception is thrown:
```
software.amazon.awssdk.services.lambda.model.LambdaException: The request signature we calculated does not match the signature you provided. Check your AWS Secret Access Key and signing method. Consult the service documentation for details.
```

AWS Java SDK version used: 2.5.17

Please take a look at this issue: https://github.com/spring-projects/spring-boot/issues/16336
In above issue, aws sdk 2.5.15 is used but this is also reproduced on 2.5.17.

**Comment by Paramood** [comment]
Also reproduced on aws sdk 1.11.522

**Comment by zoewangg** [comment]
Thanks for reporting. This is the same issue as https://github.com/aws/aws-sdk-java/issues/1919. The root cause is `org.apache.httpcomponents:httpclient:4.5.7` introduced a breaking change. 
Are you able to lock down `httpclient` version to `4.5.6`?

**Comment by Paramood** [comment]
> Are you able to lock down `httpclient` version to `4.5.6`?

Yes. This works.

**Comment by debora-ito** [comment]
The SDK version `2.7.0` was released, updating Apache `httpclient` to 4.5.9.

Please try it out and feel free to reopen this if you're still experiencing the issue.
